todolist/models.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+from django.urls import reverse
+
+
+class toDoList(models.Model):
+    title = models.CharField(max_length=255, verbose_name="Header")
+    content = models.TextField(blank=True, verbose_name="Content")
+    photo = models.ImageField(upload_to="photos/%Y/%m/%d/", verbose_name="Photo")
+    time_create = models.DateTimeField(auto_now_add=True, verbose_name="Time of creation")
+    time_update = models.DateTimeField(auto_now=True, verbose_name="Update time")
+    is_published = models.BooleanField(default=True, verbose_name="On portal")
+    cat = models.ForeignKey('Category', on_delete=models.PROTECT, null= True)
+    def __str__(self):
```

Comment:
I recommend to you use `black`. It is a code formatter which will reformat your code to a nice-looking appearance.

Reply:
You also may apply pre-commit hooks https://pre-commit.com/. It is a standard in my company.
I offer to use my config as an example https://github.com/borisevich-a-v/lang_channel/blob/master/.pre-commit-config.yaml